--- vendor/twitter/twitter-text-conformance/validate.yml	2013-07-04 11:15:55.000000000 +0900
+++ validate.yml.travis	2013-10-05 17:14:25.000000000 +0900
@@ -227,10 +227,6 @@
       text: "Test http://test.com test http://test.com test.com test"
       expected: 83
 
-    - description: "Count unicode chars outside the basic multilingual plane (double word)"
-      text: "𐀀􏿿"
-      expected: 2
-
     - description: "Count unicode chars inside the basic multilingual plane"
       text: "저찀쯿쿿"
       expected: 4
